kernels/portable/cpu/util/dtype_util.h

```diff
@@ -285,6 +289,29 @@ store_compute_to_tensor_fn<CTYPE_COMPUTE> get_store_compute_to_tensor_fn(
   return nullptr;
 }
 
+#ifndef EXECUTORCH_SELECTIVE_BUILD_DTYPE
+inline constexpr const char kGenericElementwiseOpName[] =
```

Comment:
marking this `inline` was needed for size (it changes the linkage for this, otherwise it's duplicated across translation units) and is a difference from the first attempt.